src/common/constants/constants.js

```diff
@@ -128,7 +128,7 @@ export const ERROR_CODES = {
   inactiveExpenseClass: 'inactiveExpenseClass',
   budgetExpenseClassNotFound: 'budgetExpenseClassNotFound',
   lockCalculatedTotalsMismatch: 'lockCalculatedTotalsMismatch',
-  organizationNotAVendor: 'organizationNotAVendor',
+  organizationIsNotVendor: 'organizationIsNotVendor',
```

Comment:
the whole point of this mapping to not change the keys. You can change the value only

Reply:
In this case, keys and values must be equal because they are using [here](https://github.com/folio-org/ui-invoice/blob/master/src/invoices/InvoiceDetails/utils.js#L64) for getting code and [here](https://github.com/folio-org/ui-invoice/blob/master/src/invoices/InvoiceDetails/utils.js#L76) for getting translation key.

Reply:
раненая *ерня